docs/source/data_acquisition/DataProv-Velociraptor.rst

```diff
@@ -0,0 +1,157 @@
+The Velociraptor provider
+=========================
+
+:py:mod:`Velociraptor driver documentation<msticpy.data.drivers.local_velociraptor_driver>`
+
+The ``Velociraptor`` data provider can read Velociraptor log files
```

Comment:
Let's say offline collection log files as per https://docs.velociraptor.app/docs/offline_triage/#offline-collections.
It's a different case if collection logs are sent to a SIEM (ex: https://docs.velociraptor.app/artifact_references/pages/elastic.flows.upload/, https://docs.velociraptor.app/artifact_references/pages/splunk.flows.upload/) which is likely the next step.